openmoc/plotter.py

```diff
@@ -78,12 +78,12 @@ def plot_tracks(track_generator):
   num_azim = track_generator.getNumAzim()
   spacing = track_generator.getTrackSpacing()
   num_tracks = track_generator.getNumTracks()
-  coords = track_generator.retrieveTrackCoords(num_tracks*4)
+  coords = track_generator.retrieveTrackCoords(num_tracks*6)
```

Comment:
I really think that this "6" should be defined as a variable to make it transparent